src/WebClient.ts

```diff
@@ -430,10 +430,17 @@ export class WebClient extends EventEmitter {
    */
   public readonly chat = {
     delete: (this.apiCall.bind(this, 'chat.delete')) as Method<methods.ChatDeleteArguments>,
+    deleteScheduledMessage:
+      (this.apiCall.bind(this, 'chat.deleteScheduledMessage')) as Method<methods.ChatDeleteScheduledMessagesArguments>,
```

Comment:
```suggestion
      (this.apiCall.bind(this, 'chat.deleteScheduledMessage')) as Method<methods.ChatDeleteScheduledMessageArguments>,
```

removing the `s` since the method name isn't plural.